app/cypress/support/commands.js

```diff
@@ -8,22 +8,33 @@ import getClientConfig from '$shared/utils/getClientConfig'
 
 const cache = {}
 
+const getWallet = (mnemonic) => {
+    const hdwallet = hdkey.fromMasterSeed(mnemonicToSeedSync(mnemonic))
+    return hdwallet.derivePath("m/44'/60'/0'/0").getWallet()
+}
+
 const generatePrivateKey = (mnemonic) => {
     if (!cache[mnemonic]) {
-        const hdwallet = hdkey.fromMasterSeed(mnemonicToSeedSync(mnemonic))
-        const wallet = hdwallet.derivePath("m/44'/60'/0'/0").getWallet()
-
+        const wallet = getWallet(mnemonic)
         cache[mnemonic] = wallet.getPrivateKey().toString('hex')
     }
-
     return cache[mnemonic]
 }
 
+const getAddress = (mnemonic = 'tester one') => {
+    if (!cache[mnemonic]) {
+        generatePrivateKey(mnemonic)
```

Comment:
This `if` doesn't seem like it's needed?

Reply:
True, it's prematurely cached. I'll remove it.